src/ReaderScreen/utils/index.js

```diff
@@ -1,5 +1,186 @@
-import { fontColorForReader, fontSizeForReader } from "./util";
+import { Platform } from "react-native";
 import htmlTemplate from "./gutkahtml";
 import script from "./gutkaScript";
+import { errorHandler, FallBack, colors, constant, baseFontSize } from "../../common";
 
-export { fontColorForReader, fontSizeForReader, htmlTemplate, script };
+export const fontColorForReader = (header, nightMode, text) => {
+  const { HEADER_COLOR_1_DARK, HEADER_COLOR_1_LIGHT, WHITE_COLOR, NIGHT_BLACK } = colors;
+
+  const getHeaderColor1 = () => (nightMode ? HEADER_COLOR_1_DARK : HEADER_COLOR_1_LIGHT);
+  const getHeaderColor2 = () => (nightMode ? WHITE_COLOR : NIGHT_BLACK);
+
+  const defaultColor = getHeaderColor2();
+  const gurmukhiMapping = {
+    1: getHeaderColor1(),
+    2: defaultColor,
+    6: defaultColor,
+    default: defaultColor,
+  };
+
+  const colorMapping = {
+    [constant.GURMUKHI]: gurmukhiMapping,
+    [constant.TRANSLITERATION]: getHeaderColor1(),
+    [constant.TRANSLATION]: defaultColor,
+  };
+
+  const color = colorMapping[text];
+  if (typeof color === "object") {
+    return color[header] || color.default;
+  }
+
+  return color || null;
+};
+
+export const fontSizeForReader = (SIZE, header, transliteration) => {
```

Comment:
@amitojsingh veerji, 
`SIZE` is the only parameter in full caps, any specific reason for this?

Reply:
Also, there is another variable, in small caps `size`. Can you use different name for parameter or the variable?

Reply:
fixed veer ji.